cvat-ui/src/components/annotation-page/standard-workspace/styles.scss

```diff
@@ -8,10 +8,6 @@
     height: 100%
 }
 
-.cvat-canvas-container {
-    background-color: $background-color-1;
-}
-
```

Comment:
We might stay this color by default. Isn't it?

Reply:
@bsekachev, now we set color from js. So i guess it is excessively.